src/main/java/org/sigmah/server/servlet/exporter/template/IndicatorEntryExcelTemplate.java

```diff
@@ -207,17 +209,74 @@ private void createDetailSheet(final IndicatorDTO indicator) {
 		sheetEx.addMergedRegion(new CellRangeAddress(rowIndex, rowIndex, 1, numbOfCols));
 
 		// data entry
+        //for year above months in indicators
+		int startval=2;
+		int endval=0;
+		String merge="hello";
+				row = sheetEx.createRow(++rowIndex);
+				row.setHeightInPoints(defHeight);
+				int cellIndex = 0;
+				utils.putHeader(row,++cellIndex, "Year");
+				Map<String, Integer> columnIndexMapyr=new HashMap<String, Integer>();
+				for(PivotTableData.Axis axis:leavesyear){
+					utils.putHeader(row,++cellIndex, axis.getLabel());
+					columnIndexMapyr.put(axis.getLabel(),cellIndex);
+					if(cellIndex>3){
+					if(merge.equals(axis.getLabel())){
+						
+					}
+					else{
+						endval=cellIndex-1;
+						sheetEx.addMergedRegion(new CellRangeAddress(rowIndex, rowIndex, startval, endval));
+						startval=cellIndex;
+					}
+					}
+					sheetEx.addMergedRegion(new CellRangeAddress(rowIndex, rowIndex, startval, cellIndex));
+					merge=axis.getLabel();
+					
+				}
 		// header
+        int month=0;
```

Comment:
@manishvishnoi2 : This needs an indentation fix before the pull request can be merged.